entrypoint.sh

```diff
@@ -56,6 +57,11 @@ git add .
 if git status | grep -q "Changes to be committed"
 then
   git commit --message "Update from https://github.com/$GITHUB_REPOSITORY/commit/$GITHUB_SHA"
+  echo "Closing existing PRs"
+  gh pr list --search "is:pr is:open $INPUT_DESTINATION_HEAD_BRANCH_PREFIX in:title"| while IFS= read -r line; do
+    prid=$(echo $line | awk '{print $1;}')
+    gh pr close $prid
+  done
   echo "Pushing git commit"
   git push -u origin HEAD:$INPUT_DESTINATION_HEAD_BRANCH
   echo "Creating a pull request"
```

Comment:
I was just trying to understand this code since I haven't really reviewed it before and one small comment is that using the verbose option flags would probably be helpful for viewers of this in the future vs the short hand which I had to go look up. Just my 2 cents, take it or leave it :) 

```sh
  gh pr create --title $INPUT_DESTINATION_HEAD_BRANCH \
               --body $INPUT_DESTINATION_HEAD_BRANCH \
               --base $INPUT_DESTINATION_BASE_BRANCH \
               --head $INPUT_DESTINATION_HEAD_BRANCH \
                  $PULL_REQUEST_REVIEWERS
```

Reply:
I snaked this from another dude's repo but I agree for readability it would be better. I'll add the change before I merge.